hack/test-integration.sh

```diff
@@ -69,6 +69,7 @@ os::log::start_system_logger
 function exectest() {
 	echo "Running $1..."
 
+	export TEST_ETCD_DIR=$(mktemp ${TMPDIR:-/tmp}/etcd-XXXXXXXX)
```

Comment:
You mean when run locally? We have the "clobber-at-start" paradigm everywhere else in our scripts. don't see why it won't be useful here.

*Especially* if we want to keep it after a failure (on Jenkins) we want to be able to point to a specific dir in the artifact scrubbing/download step